doc/rst/developer/bestPractices/GeneratedCode.rst

```diff
@@ -268,15 +293,18 @@ Tracing memory use
 ------------------
 
 See:
-      $CHPL_HOME/doc/rst/usingchapel/executing.rst
+     ``$CHPL_HOME/doc/rst/usingchapel/executing.rst``
 
 
 Miscellanea
```

Comment:
These scripts are mentioned in 2 other files. I suggest that we drop them from this file.